src/script/components/TitleBar/index.tsx

```diff
@@ -0,0 +1,226 @@
+  }, [callState, conversationEntity]);
+
+  const showCallControls = conversationEntity && ConversationFilter.showCallControls(conversationEntity, hasCall);
+
+  const isActivatedAccount = userState.isActivatedAccount();
+
+  const supportsVideoCall = conversationEntity?.supportsVideoCall(callingRepository.supportsConferenceCalling);
+
+  const isVideoCallingEnabled = teamState.isVideoCallingEnabled();
+
+  const conversationSubtitle =
+    conversationEntity.is1to1() && conversationEntity.firstUserEntity?.()?.isFederated
+      ? conversationEntity.firstUserEntity()?.handle ?? ''
+      : '';
+
+  const shortcut = Shortcut.getShortcutTooltip(ShortcutType.PEOPLE);
+  const peopleTooltip = t('tooltipConversationPeople', shortcut);
+
+  const onClickCollectionButton = () => {
+    amplify.publish(WebAppEvents.CONTENT.SWITCH, ContentViewModel.STATE.COLLECTION);
+  };
+
+  const onClickDetails = () => {
+    showConversationDetails(false);
+  };
+
+  return (
+    <ul id="conversation-title-bar" className="conversation-title-bar">
+      {conversationEntity && (
+        <>
+          <li className="conversation-title-bar-library">
+            {isActivatedAccount && (
+              <button
+                className="conversation-title-bar-icon icon-search"
+                type="button"
+                title={t('tooltipConversationSearch')}
+                aria-label={t('tooltipConversationSearch')}
+                onClick={onClickCollectionButton}
+                data-uie-name="do-collections"
+              >
+                <span className="visually-hidden">{t('tooltipConversationSearch')}</span>
+              </button>
+            )}
+          </li>
+
+          <li className="conversation-title-bar-name">
+            <div
+              id="show-participants"
+              onClick={onClickDetails}
+              title={peopleTooltip}
+              aria-label={peopleTooltip}
+              onKeyDown={e => handleKeyDown(e, onClickDetails)}
+              data-placement="bottom"
+              role="button"
+              tabIndex={0}
+              data-uie-name="do-participants"
+            >
+              <div className="conversation-title-bar-name-label--wrapper">
+                {conversationEntity.hasLegalHold() && (
+                  <LegalHoldDot
+                    dataUieName="status-legal-hold-conversation"
+                    className="conversation-title-bar-legal-hold"
+                    legalHoldModal={legalHoldModal}
+                    conversation={conversationEntity}
+                  />
+                )}
+
+                {conversationEntity.verification_state() === ConversationVerificationState.VERIFIED && (
+                  <Icon.Verified className="conversation-title-bar-name--verified" />
+                )}
+
+                <h2 className="conversation-title-bar-name-label" data-uie-name="status-conversation-title-bar-label">
+                  {conversationEntity.display_name()}
```

Comment:
This is an observable and show be wrapped in a `useKoSubscribableChildren` hook (this way we will get re-render when the name of the conversation changes)